api/app/tests/apps/email_integrations/test_mail.py

```diff
@@ -356,10 +381,71 @@ def test_links_environment_env_var_not_set(self):
         # we want a history entry when a email was sent
         self.assertEqual(Note.objects.count(), len(env_fe_mapping))
 
+    def test_send_mail_reporter_created_send_mail_false(self):
+        # Is the intended rule activated?
+        actions = self._get_mail_rules(['Send mail signal created'])._get_actions(self.signal)
+        self.assertEqual(len(actions), 1)
+
+        ma = MailActions(mail_rules=SIGNAL_MAIL_RULES)
+        activated = ma._get_actions(self.signal)
+        self.assertEqual(set(actions), set(activated))
+
+        # Check mail contents
+        ma.apply(signal_id=self.signal.id, send_mail=False)
+
+        # No mail should be sent
+        self.assertEqual(len(mail.outbox), 0)
+
+        # we want a history entry when a email was sent
```

Comment:
Rephrase this comment, we want no entry because nothing was sent. Current comment is confusing.